components/expenses/graphql/fragments.ts

```diff
@@ -532,6 +532,12 @@ export const expensesListFieldsFragment = gql`
           currency
         }
       }
+      ... on AccountWithHost {
+        host {
+          id
+          slug
+        }
+      }
```

Comment:
The ExpenseSummary is using this field to check if the user `isViewingExpenseInHostContext` to determine what process buttons to display. 

If not including this the buttons will swap once the full expense data is loaded (going from having the expense in list data available).

Could also send down the `host` prop from the ExpensesList to the Drawer and pass it along - but seeing as we're using loaders I'm thinking the performance hit should be negligible and that this is cleaner?